ToolTip.ios.js

```diff
@@ -24,15 +24,19 @@ var propTypes = {
 var ViewClass = React.createClass({
   getDefaultProps: function() {
     return {
-      arrowDirection: 'down'
+      arrowDirection: 'down',
+      onHide: () => true,
+      onShow: () => true
     };
   },
 
   showMenu: function() {
     ToolTipMenu.show(findNodeHandle(this.refs.toolTipText), this.getOptionTexts(), this.props.arrowDirection);
+    this.props.onShow();
```

Comment:
That's because the either onLongPress or either onPress is overwritten and there is no way to opt-out. So it can't be used like a controlled component. Hope this helps.